pkgs/os-specific/linux/openrazer/driver.nix

```diff
@@ -0,0 +1,37 @@
+{ coreutils
+, fetchFromGitHub
+, kernel
+, stdenv
+, utillinux
+}:
+
+let
+  common = import ../../../development/python-modules/openrazer/common.nix { inherit stdenv fetchFromGitHub; };
```

Comment:
Yeah this feels wrong.  Any better way to share common.nix between `os-specific` and `python-modules`?